### ISSUE TITLE
Build issue on Alpine with 13-latest-develop branch

### ISSUE THREAD

**Issue by gmr** [pganalyze/libpg_query#86]
tldr; Alpine Linux uses [musl-libc](https://www.musl-libc.org/) instead of glibc and has a package for `libexecinfo`.  `-lexecinfo` needs to be appended to `LDFLAGS` for the project to compile/link cleanly.

When I do a fresh checkout of libpg_query against the `13-latest-develop` branch and attempt to compile the project, I get the following error when linking:

```
ar: creating libpg_query.a
cc -I. -g -o examples/simple -g examples/simple.c libpg_query.a -pthread
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: libpg_query.a(src_backend_utils_error_elog.o): in function `set_backtrace':
/pgparse/libpg_query/src/postgres/src_backend_utils_error_elog.c:829: undefined reference to `backtrace'
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /pgparse/libpg_query/src/postgres/src_backend_utils_error_elog.c:830: undefined reference to `backtrace_symbols'
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: libpg_query.a(src_backend_utils_error_assert.o): in function `ExceptionalCondition':
/pgparse/libpg_query/src/postgres/src_backend_utils_error_assert.c:59: undefined reference to `backtrace'
/usr/lib/gcc/x86_64-alpine-linux-musl/10.2.1/../../../../x86_64-alpine-linux-musl/bin/ld: /pgparse/libpg_query/src/postgres/src_backend_utils_error_assert.c:60: undefined reference to `backtrace_symbols_fd'
collect2: error: ld returned 1 exit status
make: *** [Makefile:171: examples/simple] Error 1
```

`execinfo` is obviously being imported in Postgres, so I checked `configure` in the Postgres repo and found https://github.com/postgres/postgres/blob/master/configure#L11877

By setting `TEST_LDFLAGS=-lexecinfo` in my shell, it compiles cleanly and passes all tests.

I'm going to try to find a work around in my build process, but ideally `libpg_query` would be smart enough to work around this as well.

If you're looking for a PR, and want to point me in the right direction for how you'd like it addressed, I'm happy to see if I can figure it out.

**Comment by gmr** [comment]
meh, realizing this is happening when it's linking the examples and not when it's creating the lib itself, so it's a downstream issue.

**Comment by lfittl** [comment]
@gmr Yeah, thats correct, since the static library itself can't really link to execinfo, you'd have to include it in your program's LDFLAGS (like you did for the tests).

That said, I do wonder if we can avoid this problem altogether, since I don't think this particular Postgres feature is really needed in a parser library. I assume the same issue doesn't happen in `10-latest`?

**Comment by gmr** [comment]
I did not see a similar problem in `10-latest`

**Comment by lfittl** [comment]
@gmr Thanks - let me see if I can figure out how we can avoid this dependency, I'm sure you won't be the last to run into this. Will update here once I figure something out.

**Comment by lfittl** [comment]
@gmr This should be resolved now in the 13-2.0.0 release.

**Comment by gmr** [comment]
Thanks!
